### ISSUE TITLE
includedByState filter not fully working

### ISSUE THREAD

**Issue by cyrilf** [angular-ui/ui-router#1735]
`includedByState` improvement proposition

## tldr;

This works:

`<div ng-show="'**.foo.**' | includedByState">Work</div>`

and this don't.. (doesn't respect the second parameter, shown no matter what `id` is)

`<div ng-show="'**.foo.**' | includedByState:{id: 'new'}">Do not work</div>`

## Long version

The downside with it is that it doesn't use all the params from the `$state.includes` function, it only cares about the first one, which is `fullOrPartialStatename` (as reported in the [doc](http://angular-ui.github.io/ui-router/site/#/api/ui.router.state.filter:includedByState))

Here is a [plunker](http://plnkr.co/edit/tbvpxc8kkseoJOYBQXr4?p=preview) to show it live. In this example you can see that the red message is shown, no matter the params passed to the filter.

---

The issue come from [here in the code](https://github.com/angular-ui/ui-router/blob/master/src/stateFilters.js#L29) as it uses only a single param.

To resolve this, we can easily tweak the method to something like this: [Gist](https://gist.github.com/cyrilf/689dec37dd71c446e788), where `params` and `options` are given to the other method.

I may open a PR soon, but let me know **what you think about it?**

**Comment by eddiemonge** [comment]
https://gist.github.com/eddiemonge/f6a58169c2846731a1eb

**Comment by cyrilf** [comment]
Hey @eddiemonge 
I can understand that this isn't the most important issue. However, I think this is a quick win because I've already made the PR, and this isn't a breaking change. So I'll be very happy if you reconsider it.

(FYI, I've seen that the same issue will occurs in the 1.0 branch too, so I may push a PR soon if you want)

**Comment by eddiemonge** [comment]
Keeping the PR open in case @christopherthielen wants to pull it in to 1.0

**Comment by christopherthielen** [comment]
On mobile but this seems ok to me.

**Comment by christopherthielen** [comment]
Link the PR?

**Comment by cyrilf** [comment]
The PR for this is [here](https://github.com/angular-ui/ui-router/pull/1736)
I didn't took the time to make one for the 1.0, but I may do it soon.

**Comment by cyrilf** [comment]
Ok, so here you are for the [PR](https://github.com/angular-ui/ui-router/pull/2193) to the `feature-1.0`
